src/main/java/org/fundacionjala/coding/kevincristhian/CesarCrypto.java

```diff
@@ -0,0 +1,139 @@
+package org.fundacionjala.coding.kevincristhian;
+
+/**
+ * class for cesar and vigenere methods.
+ */
+public class CesarCrypto {
+    private static final int NINETY = 90;
+    private static final int SIXTY_FOUR = 64;
+    private static final int TWENTY_SIX = 26;
+    /**
+     * methid which change the key
+     * between -26,26.
+     * @param key original value.
+     * @return int.
+     */
+    private int verifyKey(int key) {
+        if (key % TWENTY_SIX == 0) {
+            return 0;
+        }
+        while (Math.abs(key) > TWENTY_SIX) {
+                if (key > 0) {
+                    key -= TWENTY_SIX;
+                } else {
+                    key += TWENTY_SIX;
+                }
+        }
+        return key;
+    }
+
+    /**
+     * create the key sentence with the key word.
+     * @param message message to encodeVigenere.
+     * @param key word.
+     * @return string.
+     */
+    private String createKeyString(final String message, final String key) {
+        int posKey = 0;
+        int sizeKey = key.length() - 1;
+        StringBuffer newKey = new StringBuffer();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace the synchronized class "StringBuffer" by an unsynchronized one such as "StringBuilder". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1149)